packages/core/contracts/oracle/implementation/OptimisticOracle.sol

```diff
@@ -92,7 +92,8 @@ contract OptimisticOracle is OptimisticOracleInterface, Testable, Lockable {
         bytes32 identifier,
         uint256 timestamp,
         bytes ancillaryData,
-        int256 proposedPrice
+        int256 proposedPrice,
+        uint256 expirationTimestamp
```

Comment:
New contract event param that makes the OO client a lot easier to implement (with fewer web3 calls)

Reply:
Your changes here do not affect the internal state of the contract in any way, so I think this is probably okay. Probably would be good to just shoot this over to OZ when this gets merged.